### DISCUSSION TITLE
Add more data to show

### DISCUSSION THREAD

**Discussion by AlexJacu** [stephan192/lhpapi#3] [Ideas]
I would suggest, also to use the following data to show more Information to enable the user to identify the actual status.
Following Data would be very helpful:
Meldestufe 1	85 cm
Meldestufe 2	120 cm
Meldestufe 3	180 cm
HHW (Höchster jemals gemessener Hochwasserstand)	200 cm
MNQ (Mittlerer Niedrigwasserabfluss)	0,20 m³/s
MQ (Mittlerer Abfluss)	0,85 m³/s
MHQ (Mittlerer Hochwasserabfluss)	11,6 m³/s
HHQ (Höchster jemals gemessener Durchfluss)	34,8 m³/s

Example: https://www.hlnug.de/static/pegel/wiskiweb3/webpublic/#/overview/Wasserstand/station/42059/Eppstein/stationInfoHlnug

**Comment by stephan192** [comment]
Hi @AlexJacu, i decided to not add the listed values to the api because of the following reasons:
1. They are static values which normally don't change. If someone really needs them one can look them up once on the pegel's page.
2. Only a few portals deliver these values and i want to make an api which is widely the same for all state portals.
3. Finally: These values are not required to identify the actual status. They provide no additional information because the "Meldestufe" as number (0-4) already includes the final and most imported information for the user.
